app/views/mailer/application_confirmation_email.html.erb

```diff
@@ -1,21 +1,13 @@
-<h2 style="margin: 0px 0px 10px;">Thanks for applying to BrickHack!</h2>
-<span class="orange-block" style="background: #F98728;display: block;height: 8px;margin-bottom: 35px;width: 200px;"></span>
-<p style="margin: 0 0 8px;">We've successfully received your application to BrickHack 2015. Below is a copy of your information.</p>
-<p style="margin: 0 0 8px;">If necessary, you may update your application at <a href="<%= edit_questionnaires_url(host: "brickhack.io") %>" style="color: #F98728;"> <%= edit_questionnaires_url(host: "brickhack.io") %></a></p>
-<h4 style="margin: 0px 0px 10px;">You will be notified of your acceptance status some time before April 18th.</h4>
-<ul style="list-style: none;padding: 0;">
-  <li style="margin:5px 0px;">Name: <%= @questionnaire.full_name %></li>
-  <li style="margin:5px 0px;">Email: <%= @questionnaire.email %></li>
-  <li style="margin:5px 0px;">Phone: <%= @questionnaire.phone %></li>
-  <li style="margin:5px 0px;">Location: <%= @questionnaire.full_location %></li>
-  <li style="margin:5px 0px;">Birthday: <%= @questionnaire.birthday_formatted %></li>
-  <li style="margin:5px 0px;">School: <%= @questionnaire.school.name %></li>
-  <li style="margin:5px 0px;">Year: <%= Questionnaire::POSSIBLE_YEARS[@questionnaire.year] %></li>
-  <li style="margin:5px 0px;">Interest: <%= @questionnaire.interest %></li>
-  <li style="margin:5px 0px;">Experience: <%= Questionnaire::POSSIBLE_EXPERIENCES[@questionnaire.experience] %></li>
-  <li style="margin:5px 0px;">Shirt Size: <%= @questionnaire.shirt_size %></li>
-  <li style="margin:5px 0px;">Resume: <%= @questionnaire.resume? ? "Provided" : "Not provided" %></li>
-  <li style="margin:5px 0px;">Dietary & Medical Notes: <%= @questionnaire.dietary_medical_notes || "None" %></li>
-  <li style="margin:5px 0px;">Portfolio Link: <%= @questionnaire.portfolio_url? ? @questionnaire.portfolio_url : 'Not provided' %></li>
-  <li style="margin:5px 0px;">GitHub/BitBucket Link: <%= @questionnaire.vcs_url? ? @questionnaire.vcs_url : 'Not provided' %></li>
-</ul>
+<h2 class="center">
+  Thanks for Applying!
+</h2>
+
+<p>
+  Hey <%= @questionnaire.first_name %>, we've received your application to BrickHack 2016!  You'll be notified of your acceptance status around May.  In the meantime, you can review and edit your information by clicking the button below.
+</p>
+
+<%= link_to 'http://mlh.io', class: 'nodecor' do %>
```

Comment:
This will likely be a link to BrickHack's website, but that can be updated when the API merge is ready.